packages/babel-plugin-minify-dead-code-elimination/__tests__/fixtures/unused-import-bindings/expected.js

```diff
@@ -0,0 +1,6 @@
+import "file";
+import "test-variable-names";
+import "weird-test-variable-names";
+import bark from "dogs";
+import { woof } from "dogs";
```

Comment:
Should these two be combined?

```js
import bark, { woof } from "dogs";
```

Reply:
Yes, but that's trickier. If we want to support it I'd rather do it was a separate PR.